packages/camera/camera_web/lib/src/camera.dart

```diff
@@ -638,4 +643,74 @@ class Camera {
       ..height = '100%'
       ..objectFit = 'cover';
   }
+
+  final StreamController<CameraImageData> _cameraFrameStreamController =
+      StreamController<CameraImageData>.broadcast();
+
+  // TODO(replace): introduced fps in
+  /// [CameraImageStreamOptions]
+  final int cameraStreamFPS = 30;
+
+  /// Returns a stream of camera frames.
+  ///
+  /// To stop listening to new animation frames close all listening streams.
+  Stream<CameraImageData> cameraFrameStream({
+    CameraImageStreamOptions? options,
+  }) {
+    _cameraFrameStreamController.onListen = () {
+      _triggerAnimationFramesLoop(
+        _addCameraImageDataEvent,
+        fps: cameraStreamFPS,
+      );
+    };
+
+    return _cameraFrameStreamController.stream;
+  }
+
+  /// Triggers animation frames in a loop as long as
+  /// [_cameraFrameStreamController.hasListener] and executes the callback
+  void _triggerAnimationFramesLoop(
+    VoidCallback action, {
+    required int fps,
+  }) {
+    Completer<void> completer = Completer<void>();
+    completer.complete();
```

Comment:
Why do we need this? Remove?

Reply:
This is fixed, can be marked as resolved?